src/app/_components/PostItem.tsx

```diff
@@ -22,10 +22,16 @@ const convertOptionsToString = (options: string[]) => {
     result.push(option);
   }
 
-  if (remainingCount > 0)
-    return `${result.join(',')} 외 ${remainingCount}개 투표 진행 중`;
+  const optionString = result.join(', ');
+  let suffix = remainingCount > 0 ? ` 외 ${remainingCount}개 ` : ' ';
 
-  return `${result.join(',')} 투표 진행 중`;
+  if (isTerminated) {
+    suffix += '투표 종료';
+  } else {
+    suffix += '투표 진행 중';
+  }
```

Comment:
```suggestion
const text = `${remainingCount}개 ${isTerminated ? "투표종료" : "투표진행중"}`
```

 처럼 인라인으로 분기처리 다 해서 넣어주세요 !